clusterman/config.py

```diff
@@ -86,6 +89,32 @@ def load_cluster_pool_config(cluster: str, pool: str, scheduler: str, signals_br
         )
 
 
+def get_pool_config_path_if_exists(cluster, pool, scheduler):
+    """Checks that a config file for a pool scheduled by a specific scheduler
+    exists in a cluster. If it does, return the path for the file. If not,
+    exit.
+    """
+    pool_config_file = get_pool_config_path(cluster, pool, scheduler)
+    if os.path.exists(pool_config_file):
+        return pool_config_file
+    else:
+        cluster_dir = get_cluster_config_directory(cluster)
+        if os.path.exists(cluster_dir):
+            other_sched_files = glob.glob(os.path.join(cluster_dir, f'{pool}.*'))
+            if len(other_sched_files) > 0:
+                schedulers = [os.path.splitext(f)[1][1:] for f in other_sched_files]
+                msg = (
+                    f"Pool '{pool}' is scheduled in cluster '{cluster}' by {schedulers}, "
+                    f"not '{scheduler}'"
+                )
+            else:
+                msg = f"Pool '{pool}' does not exist in cluster '{cluster}'"
+        else:
+            msg = f"Cluster '{cluster}' does not exist"
+        print(Fore.RED + msg + Style.RESET_ALL)
+        raise SystemExit
```

Comment:
Can we raise something other than `SystemExit` here?

Reply:
This function is only ever called during start up so if something fails, the process needs to fail. To make it a little cleaner, I've changed it to `sys.exit`.